easybuild/easyconfigs/n/netcdf4-python/netcdf-python-1.4.1-CrayGNU-18.08-python2.eb

```diff
@@ -0,0 +1,33 @@
+#
+name = 'netcdf-python'
+version = '1.4.1'
+
+homepage = 'http://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+
+source_urls = ['https://pypi.python.org/packages/source/n/netCDF4/']
+sources = ['netCDF4-%(version)s.tar.gz']
+
+req_py_majver = 2
+req_py_minver = 7
+
+pythonver = '%s.%s' % (req_py_majver, req_py_minver)
+versionsuffix = '-python%s' % req_py_majver
+
+dependencies = [
+    ('cray-python/2.7.15.1', EXTERNAL_MODULE),
+    ('netCDF-Fortran', '4.4.4'),
+    ('cftime', '1.0.0', '-python2'),
+    ('cray-hdf5/1.10.0.3', EXTERNAL_MODULE),
```

Comment:
There is a newer version:
('cray-hdf5/1.10.2.0', EXTERNAL_MODULE),

Reply:
No, we cannot use it. The problem is that latest netCDF-Fortran is 4.4.4 and depends on that exact `cray-hdf5` module. However, the latest netCDF library (C bindings) is 4.6.1.